.github/workflows/build-ios.yml

```diff
@@ -0,0 +1,186 @@
+name: Build IOS
+on:
+  workflow_call:
+    inputs:
+      version_file:
+        type: string
+        required: true
+      macos_version:
+        type: string
+        required: true
+      xcode_version:
+        type: string
+        required: true
+      version:
+        type: string
+        required: true
+      prefix:
+        type: string
+        required: true
+env:
+  GOPRIVATE: github.com/getlantern
+  S3_BUCKET: lantern
+jobs:
+  build-ios:
+    runs-on: ${{ inputs.macos_version }}
+    permissions:
+      contents: "read"
+      id-token: "write"
+    env:
+      version: ${{ inputs.version }}
+      version_file: ${{ inputs.version_file }}
+      prefix: ${{ inputs.prefix }}
+      AC_USERNAME: accounts@getlantern.org
+      AC_PASSWORD: ${{ secrets.AC_PASSWORD }}
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          lfs: true
+      - name: Pull LFS objects
+        run: git lfs pull
+
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: "go.mod"
+
+      - name: Setup Xcode
+        uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: ${{ inputs.xcode_version }}
+
+      - name: Repo access
+        run: |
+          mkdir /tmp/cache
+          echo "machine github.com login ${{ secrets.GH_TOKEN }} password x-oauth-basic" > /tmp/cache/.netrc
+          chmod 600 /tmp/cache/.netrc
+
+      - name: Setup Sentry CLI
+        uses: mathieu-bour/setup-sentry-cli@v1
+        with:
+          version: latest
+          token: ${{ SECRETS.SENTRY_TOKEN }} # from GitHub secrets
+          organization: getlantern
+          project: lantern-ios
+
+      - name: Install Flutter
+        uses: subosito/flutter-action@v2
+        with:
+          channel: "stable"
+          flutter-version-file: pubspec.yaml
+
+      - name: Install the Apple certificate and provisioning profile
+        env:
+          BUILD_CERTIFICATE_BASE64: ${{ secrets.IOS_CERTIFICATE_P12_BASE64 }}
```

Comment:
Are these secrets in 1Password? If not, maybe add them there and link to them in the README? You could also mention what they represent, and how you obtain the value of each. For instance..

- BUILD_CERTIFICATE_BASE64: Apple Distribution Certificate, Base64 -  https://developer.apple.com/account/resources/certificates/add

Reply:
All files, passwords, and certificates are already in 1pass.

Reply:
@jigar-f Where? I just searched for "BUILD_PROVISION_PROFILE_BASE64" and "IOS_PROVISION_PROFILE_BASE64" and I'm not finding anything.

Reply:
@atavism Search for this **IOS Certificated and profiles**

Reply:
@jigar-f Thanks! Found it.